src/Component/Button/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactNode } from "react";
+import "./button.css";
+export interface buttonProp {
+  type: String;
+  children: ReactNode;
+  size: String;
+  onclick: () => void;
+}
+export const Button: React.FC<buttonProp> = ({
+  children,
+  type = "primary",
+  size = "medium",
+  onclick,
+}) => {
+  return (
+    <button className={`button ${type} ${size}`} onClick={onclick}>
```

Comment:
We will use styled components eventually

Reply:
Yeah, that's good actually I was thinking to use it. I will change this with the styled component.

Reply:
Don't change it just yet. I have created a ticket for it